matrix/common/lambda_handler.py

```diff
@@ -26,8 +32,15 @@ def invoke(self, fn_name: LambdaName, payload: dict):
         :param fn_name: The LambdaName of the function to be invoked
         :param payload: Data passed to invoked lambda
         """
-        self._client.invoke(
-            FunctionName=fn_name.value,
-            InvocationType="Event",
-            Payload=json.dumps(payload).encode(),
-        )
+        try:
+            self._client.invoke(
+                FunctionName=fn_name.value,
+                InvocationType="Event",
+                Payload=json.dumps(payload).encode(),
+            )
+        except ClientError:
+            size = len(json.dumps(payload).encore())
+            self.request_tracker.log_error(f"An error occurred while processing your request. The size of the "
```

Comment:
Probably a bit optimistic to assume that this is always the reason for `ClientError`. If you want a special message, maybe something like
```python
except ClientError as e:
    if e["Error"]["Code"] == "EntityTooLarge":
        # special log
    else:
        log_error(e["Error"]["Message"])
```

Reply:
Right and as you said, this capturing this error's kind of unnecessary. I couldn't think of another error to capture haha. I'll scrap this and handle the unreadable bundle case then